package_manifest.yaml

```diff
@@ -180,7 +180,10 @@ foreman_packages:
     rubygem-extlib: {}
     rubygem-facter: {}
     rubygem-faraday: {}
-    rubygem-fast_gettext: {}
+    rubygem-fast_gettext:
+      releasers:
+        - koji-foreman
+        - koji-foreman-plugins
```

Comment:
This will work but likely fail if later on we update the package. The repoclosure check for one would be different given this is building an SCL and non-SCL version and targeting two different repositories. Do you want to just roll with it as is or tackle this issue now?

Reply:
Guys please fix this for me so I can come up with FDI 3.5.0 build which is a *huge* overdue.